tests/AuthTest.php

```diff
@@ -63,23 +69,31 @@ public function testValidRegistration()
                 ]
             );
 
-            $this->seeJsonEquals([
-                'errors' => ['The email has already been taken.']
-            ]);
+            $this->assertResponseStatus(400);
         }
 
         public function testValidLogin()
         {
-            $this->json(
+            $profile = factory(App\Profile::class)->make();
+
+            $resp = $this->json(
                 'POST',
                 '/api/v1/app/starapi-testing/login',
                 [
-                   'email' => 'marko@marko.com',
+                    'email' => 'marko@marko.com',
```

Comment:
Where did this profile come from, create new user within the test and use that instead.